packages/callout/src/Callout.tsx

```diff
@@ -56,20 +54,19 @@ const headerIconStyle = css`
 `;
```

Comment:
Something about this positioning looks off

Reply:
@TheSonOfThomp Can you share a screenshot?

Reply:
Nevermind, you have it right 👍